packages/common/container-definitions/src/loader.ts

```diff
@@ -245,6 +245,11 @@ export interface IContainerEvents extends IEvent {
 	 * @see {@link IContainer.isDirty}
 	 */
 	(event: "saved", listener: (dirty: boolean) => void);
+
+	/**
+	 * Emitted when the some of the delta connection properties are initialized or updated.
+	 */
+	(event: "deltaConnectionUpdated", listener: (metadata?: Record<string, unknown>) => void);
```

Comment:
I do not like the name of event :)
I think (for most part) users should not care where data came from. This is data about container / file, i.e. properties about file that gets updated through some means. They could be updated through connection-specific workflows (joinSession, signals), but also maybe some data comes from TreesLatest, maybe through some other APIs. As such I'd probably remove notion to connection and call it based on what this data is about

Reply:
Same feedback for other names across code base (at least publicly exposed names)

Reply:
Since the update could be about any data as it could be extended to other usages also in future, I have kept the name a but general. Renamed it to "metadataUpdate'. Let me know, if this sounds good.